package.json

```diff
@@ -11,12 +11,13 @@
     "start": "node public/app.js",
     "clean": "rimraf public",
     "build": "npm run clean && npm run build:ts && node public/esbuild.js",
-    "lint": "npx eslint . --ignore-pattern \"node_modules/*\" --ignore-pattern \"public/*\" --fix",
+    "lint": "npx eslint . --ignore-pattern \"node_modules/*\" --ignore-pattern \"public/*\" --ignore-pattern \"test/**/*.spec.ts\" --fix",
```

Comment:
Refactor to eliminate multiple `--ignore-pattern `flags by combining patterns into a single expression.
See [this reference](https://github.com/ministryofjustice/laa-manage-your-civil-cases/pull/70/commits/c23a9136568dd9bc1d6d51ef1a3450be0ec55804) for more details.

Reply:
updated